server/server.go

```diff
@@ -173,44 +176,44 @@ func (s *server) Run(rootCtx context.Context) (err error) {
 				return
 			}
 
-			zap.L().Info("serving gRPC gateway", zap.String("port", s.cfg.GwPort))
+			s.log.Info("serving gRPC gateway at port %s", s.cfg.GwPort)
 			gwErrc <- s.gwServer.ListenAndServe()
 		}()
 	}
 
 	select {
 	case <-rootCtx.Done():
-		zap.L().Warn("root context is done")
+		s.log.Warn("root context is done")
 
 		if !s.cfg.GwDisabled {
-			zap.L().Warn("closed gRPC gateway server", zap.Error(s.gwServer.Close()))
+			s.log.Warn("closed gRPC gateway server: %w", s.gwServer.Close())
```

Comment:
I am not sure %w makes sense here, as there is not new error where to wrap into. same for other %w ocurrences in logging

Reply:
here and in other files